.eslintrc.json

```diff
@@ -14,7 +14,8 @@
         "inject": false,
         "module": false,
         "Sk": false,
-        "tie": false
+        "tie": false,
+        "tieBasic": false
```

Comment:
So, when we were working together on this, I had just used tieBasic as a placeholder to get things working. But I thought we were going to use "tie" for the common module and "tieQuestionPage", "tieMenuPage", etc. for the others.

It might be worth picking a name for the common module that's more descriptive of what it is. It's not a basic version of TIE, since it does not function on its own. Perhaps "tieCommon" (since it is the common part of all the various TIE pages), but you might have other ideas.

I'm also aware that stuff has to be renamed from "tie" to "tieQuestionPage", It's fine to do this in a separate pass or in this PR, but it should happen as part of the infrastructure update.